content/00.front-matter.md

```diff
@@ -30,18 +30,21 @@ on {{manubot.date}}.
 + **{{author.name}}**
   {% if author.corresponding is defined and author.corresponding == true -%}^[✉](#correspondence)^{%- endif -%}
   <br>
-  {%- if author.orcid is defined and author.orcid is not none %}
+  {%- if author.orcid is defined and author.orcid is not none %} {% set line1 = true %}
```

Comment:
I wonder if the purpose would be a bit clearer if we:

`set has_ids = false` prior to the if statements and then had each if condition `set has_ids = true`. Then final check would be  `{%- if has_ids == true %}`.

Very similar so let's go with whichever @miltondp thinks will be most understandable for future people looking at the jinja code.

Reply:
I like this proposed syntax change if it's okay with @miltondp.

Reply:
I think these are good changes, I'll add those.